spec.html

```diff
@@ -2593,183 +2593,120 @@ <h1>The Object Type</h1>
+                If the value is an Object it must be a function object. The function's [[Call]] internal method (<emu-xref href="#table-additional-essential-internal-methods-of-function-objects"></emu-xref>) is called with an empty arguments list to retrieve the property value each time a get access of the property is performed.
               </td>
             </tr>
             <tr>
               <td>
-                [[Enumerable]]
-              </td>
-              <td>
-                Boolean
+                [[Set]]
               </td>
               <td>
-                If *true*, the property is to be enumerated by a for-in enumeration (see <emu-xref href="#sec-for-in-and-for-of-statements"></emu-xref>). Otherwise, the property is said to be non-enumerable.
+                accessor property
               </td>
-            </tr>
-            <tr>
               <td>
-                [[Configurable]]
+                Object | Undefined
               </td>
               <td>
-                Boolean
+                *undefined*
               </td>
               <td>
-                If *false*, attempts to delete the property, change the property to be a data property, or change its attributes will fail.
+                If the value is an Object it must be a function object. The function's [[Call]] internal method (<emu-xref href="#table-additional-essential-internal-methods-of-function-objects"></emu-xref>) is called with an arguments list containing the assigned value as its sole argument each time a set access of the property is performed. The effect of a property's [[Set]] internal method may, but is not required to, have an effect on the value returned by subsequent calls to the property's [[Get]] internal method.
               </td>
             </tr>
-          </table>
-        </emu-table>
-        <p>If the initial values of a property's attributes are not explicitly specified by this specification, the default value defined in <emu-xref href="#table-default-attribute-values"></emu-xref> is used.</p>
-        <emu-table id="table-default-attribute-values" caption="Default Attribute Values" oldids="table-4">
-          <table>
             <tr>
-              <th>
-                Attribute Name
-              </th>
-              <th>
-                Default Value
-              </th>
-            </tr>
-            <tr>
-              <td>
-                [[Value]]
-              </td>
               <td>
-                *undefined*
+                [[Enumerable]]
               </td>
-            </tr>
-            <tr>
               <td>
-                [[Get]]
+                All
```

Comment:
I think "data property or accessor property" is clearer.

Reply:
Although we may also have to add "or generic property"? Still, probably an improvement.

Reply:
"generic" can only apply to a Property Descriptor; a property must only be a data or an accessor property.